src/App.ts

```diff
@@ -1286,14 +1291,15 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
         appToken,
         clientId,
         clientSecret,
-        stateSecret,
-        redirectUri,
+        customRoutes,
+        installerOptions: this.installerOptions,
         installationStore,
+        redirectUri,
+        ...socketModeOptions,
```

Comment:
@mwbrooks @WilliamBergamin

❓ `installerOptions` is made public on the instance. I'm not sure why, exactly, but we can do the same for `socketModeOptions` if we think there's a use case for exposing it.